### PR TITLE
docs: Components catalog

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/5331797912

### CODE REVIEW

File: src/storybook/stand-alone-documentaion/catalog/catalog.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Meta, Story } from "@storybook/addon-docs";
+import { Catalog } from "./Catalog/Catalog.stories";
+
+<Meta title="Catalog" />
+
+# Catalog
+
+<Story name="Catalog">
```
Comment (SergeyRoyt) [inline src/storybook/stand-alone-documentaion/catalog/catalog.stories.mdx:8]: That means that it's also captured in Chromatic snapshots

File: src/storybook/stand-alone-documentaion/catalog/assets/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import emptyStateExample from "./emptyStateExample.svg";
+
+export { emptyStateExample };
```
Comment (talkor) [inline src/storybook/stand-alone-documentaion/catalog/assets/index.js:3]: Is it possible to use svg as an image in MDX?

File: src/storybook/stand-alone-documentaion/catalog/Catalog/Catalog.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+import { RelatedComponentsDecorator } from "../../../components";
+import { descriptionComponentsWithoutGeneralDescriptionMap } from "../../../components/related-components/component-description-map";
+import { Search } from "../../../../components";
+import { CatalogEmptyState } from "../EmptyState/Catalog.stories.EmptyState";
+import styles from "./Catalog.stories.module.scss";
+
+const RELATED_COMPONENT_NAMES = Array.from(descriptionComponentsWithoutGeneralDescriptionMap.keys())
```
Comment (talkor) [inline src/storybook/stand-alone-documentaion/catalog/Catalog/Catalog.stories.tsx:8]: Nitpick: I don't think it should be constant

File: src/storybook/stand-alone-documentaion/catalog/EmptyState/Catalog.stories.EmptyState.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+export const CatalogEmptyState = () => {
+  return (
+    <div className={styles.emptyStateContainer} aria-labelledby="empty-state-id">
+      <img style={{ width: "290px" }} src={emptyStateExample} alt="" />
+      <Heading type={Heading.types.H2} id="empty-state-id" className={styles.emptyStateHeading}>
+        We haven&apos;t found this component in the catalog
+      </Heading>
+      <Text element="span" type={Text.types.TEXT1} style={{ width: "50%", textAlign: "center" }} ellipsis={false}>
+        Not all the components have been added to the Catalog already, please, also check the{" "}
+        <b>search in the left panel</b>.
```
Comment (talkor) [inline src/storybook/stand-alone-documentaion/catalog/EmptyState/Catalog.stories.EmptyState.tsx:15]: Can we also Text with weight?

File: src/storybook/stand-alone-documentaion/catalog/EmptyState/Catalog.stories.EmptyState.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { emptyStateExample } from "../assets";
+import { Heading } from "../../../../next/next";
+import { Text } from "../../../../components";
+import styles from "./Catalog.stories.EmptyState.module.scss";
+
+export const CatalogEmptyState = () => {
+  return (
+    <div className={styles.emptyStateContainer} aria-labelledby="empty-state-id">
+      <img style={{ width: "290px" }} src={emptyStateExample} alt="" />
```
Comment (talkor) [inline src/storybook/stand-alone-documentaion/catalog/EmptyState/Catalog.stories.EmptyState.tsx:9]:
```suggestion
      <img style={{ width: "290px" }} src={emptyStateExample} alt="" role="presentation" />
```

**Review by talkor — APPROVED** [review]
Looks good! Left some minor comments
Also, there's a small issue with alignment with the hero banner
Screenshot 2023-11-15 at 12 25 41

File: src/storybook/stand-alone-documentaion/catalog/Catalog/Catalog.stories.tsx (hunk 3 above)
Comment (SergeyRoyt) [reply src/storybook/stand-alone-documentaion/catalog/Catalog/Catalog.stories.tsx:8]: Why not? Why to keep it in the component if it's not changing?

Comment (talkor) [reply src/storybook/stand-alone-documentaion/catalog/Catalog/Catalog.stories.tsx:8]: It's depenedent on an external map

**Comment by talkor** [comment]
@SergeyRoyt I also noticed links are opened in a new tab, I think it should be better if it navigates to the component inside SB

File: src/storybook/stand-alone-documentaion/catalog/assets/index.js (hunk 2 above)
Comment (SergeyRoyt) [reply src/storybook/stand-alone-documentaion/catalog/assets/index.js]: Refactored this part

File: src/storybook/stand-alone-documentaion/catalog/Catalog/Catalog.stories.tsx (hunk 3 above)
Comment (SergeyRoyt) [reply src/storybook/stand-alone-documentaion/catalog/Catalog/Catalog.stories.tsx:8]: Which is constant any way

**Pull request merged**